### ISSUE TITLE
Does spotlight have tensorboard integration?

### ISSUE THREAD

**Issue by drop-ella** [maciejkula/spotlight#166]
Tensorboard is now supported by PyTorch. Does spotlight support it or will do so in the future? And what is a suggested way of monitoring model training (training and validation loss/metrics at step n) when using spotlight models?

Thank you very much!

**Comment by snemistry** [comment]
I modified spotlight implicit classes by adding notifier function for every epoch completion, so you have full control over however you can do with loss and self._net model. I use to write loss and my own eval function in tensorboard and works great.
